### DISCUSSION TITLE
v5 - npm update failing

### DISCUSSION THREAD

**Discussion by sn0rcha** [adonisjs/core#2254] [General]
Hey There,

Tried doing an "npm update" on my v5 project this morning and am receiving the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: SCCGP@1.0.0
npm ERR! Found: @adonisjs/lucid@9.0.3
npm ERR! node_modules/@adonisjs/lucid
npm ERR!   @adonisjs/lucid@"^9.0.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @adonisjs/lucid@"^10.0.0" from @adonisjs/auth@5.1.1
npm ERR! node_modules/@adonisjs/auth
npm ERR!   @adonisjs/auth@"^5.1.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

Any ideas on how to fix this?

Cheers,
Ben

**Comment by McSneaky** [comment]
What are your NPM and Node versions?

**Comment by RomainLanz** [comment]
Hey @sn0rcha! 👋 

You need to update your dependencies correctly.

Since you are using AdonisJS 5, many packages are still under the `@alpha` tag, whereas some aren't.
When running `npm update`, it will not catch up with the `@alpha`.

You have two options.

1. You use a tool like [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates) and use the `--target greatest` option to get all the changes (even `@alpha` one).
2. You re-install the package using the `@alpha` tag (i.e.: `npm i @adonisjs/lucid@alpha ...`)

**Comment by sn0rcha** [comment]
Many thanks guys, 

```npm i @adonisjs/lucid@alpha```

Did the trick. I tried using the npm-check-updates but it broke something to do with validations.

Just wanted to give you guys a big shoutout for version 5, it's an absolute pleasure to code with and the performance is amazing, awesome work!

Cheers,
Ben
